src/messages.js

```diff
@@ -65,9 +72,111 @@
   JSAV.ext.clearumsg = function(msg, options) {
     this._msg.clear();
   };
-  
+
+  ////////////////////////
+  ///////TextToSpeech/////
+  JSAV.ext.textToSpeech = function(speechText) {
+    var modifiedMsg;
+    modifiedMsg = speechText.replace(/<[^>]*>/g, "");
+    modifiedMsg = modifiedMsg.replace(/\$/g, "");
+    modifiedMsg = modifiedMsg.replace(/\\mathbf/gi, "");
+    modifiedMsg = modifiedMsg.replace(/\\displaystyle/gi, "");
+    modifiedMsg = modifiedMsg.replace(/\\mbox/gi, "");
+    modifiedMsg = modifiedMsg.replace(/n-/gi, "n minus");
+    modifiedMsg = modifiedMsg.replace(/m-/gi, "m minus");
+    modifiedMsg = modifiedMsg.replace(/%/gi, "remainder");
```

Comment:
Some of these seem quite specific to the markup the visualizations are using. I'd love to see a way for an animation to add to this list of replacements. Not needed for merge, but just something to think about in the future as this way will not scale that well.